iopipe/contrib/logger/stream.py

```diff
@@ -1,7 +1,14 @@
+import sys
+
+
 class StreamToLogger(object):
```

Comment:
Question, does this new stream replace `stdout`? If it does, should `flush()` additionally be forwarded in the event that explicit flushes want to be done?

Reply:
StreamToLogger implements stdout redirection. So in this case it overloads `write()` to send log messages to the IOpipe logger and then passes the buffer back to stdout's `write()`. The `__getattr__` effectively exposes all of stdout's methods from within StreamToLogger. So if you called StreamToLogger.flush() you're effectively calling sys.stdout.flush().